js/login.js

```diff
@@ -0,0 +1,65 @@
+async function sendAuthRequest() {
+    const username = document.getElementById('username').value;
+    const password = document.getElementById('password').value;
+
+    const data = {
+        Username: username,
+        Password: password
+    };
+
+    const requestOptions = {
+        method: 'POST',
+        headers: { 'Content-Type': 'application/json' },
+        body: JSON.stringify(data),
+    };
+
+    try {
+        const response = await fetch('http://localhost:8000/api/auth', requestOptions);
+
+        if (response.status === 202) {
```

Comment:
Add else statement for handling of other status codes

Reply:
@Kuan-Zher, I've added additional conditional else-if statements to handle other status codes already